test/lint/lint-cppcheck-dash.sh

```diff
@@ -35,6 +35,7 @@ IGNORED_WARNINGS=(
     "src/masternode/sync.cpp:.*: warning: Variable 'pnode' can be declared as pointer to const \[constVariableReference\]"
     "src/wallet/bip39.cpp.*: warning: The scope of the variable 'ssCurrentWord' can be reduced. \[variableScope\]"
     "src/.*:.*: warning: Local variable '_' shadows outer function \[shadowFunction\]"
+    "src/stats/rawsender.h:.* Struct 'RawMessage' has a constructor with 1 argument that is not explicit."
```

Comment:
pls see 6dd9b07e2b

Reply:
`RawMessage`'s constructor was intentionally kept implicit because its primary purpose is to abstract the `std::string`-to-`std::vector<uint8_t>` conversion and making that explicit seems to partially defeat the point.

Reply:
well, `static_cast<const RawMessage>` is `std::string`-to-`RawMessage`, not `std::string`-to-`std::vector<uint8_t>` 🤷‍♂️ 🙂

Reply:
or alternatively, we could maybe have
```
std::optional<std::string> RawSender::SendString(const std::string& msg_str) noexcept
{
    return Send(static_cast<const RawMessage>(msg_str));
}
```
instead so that this conversion would be hidden in RawSender implementation

Reply:
or we could do operate RawMessag-es from the start
```diff
diff --git a/src/stats/client.cpp b/src/stats/client.cpp
index bd47d96a82..71b122a1d6 100644
--- a/src/stats/client.cpp
+++ b/src/stats/client.cpp
@@ -158,13 +158,13 @@ bool StatsdClient::send(const std::string& key, T1 value, const std::string& typ
     }
 
     // Construct the message and if our message isn't always-send, report the sample rate
-    std::string buf{strprintf("%s%s%s:%f|%s", m_prefix, key, m_suffix, value, type)};
+    RawMessage msg{strprintf("%s%s%s:%f|%s", m_prefix, key, m_suffix, value, type)};
     if (!always_send) {
-        buf += strprintf("|@%.2f", sample_rate);
+        msg += RawMessage{strprintf("|@%.2f", sample_rate)};
     }
 
     // Send it and report an error if we encounter one
-    if (auto error_opt = m_sender->Send(buf); error_opt.has_value()) {
+    if (auto error_opt = m_sender->Send(msg); error_opt.has_value()) {
         LogPrintf("ERROR: %s.\n", error_opt.value());
         return false;
     }
```